partner_enforce_iban_validation/__manifest__.py

```
@@ -0,0 +1,20 @@
+    "author": "Cetmix OÜ, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": [
+        "contacts",
+        "base_iban",
+    ],
+    "data": [
+        "views/res_config_settings_view.xml",
+    ],
+    "application": False,
+    "installable": True,
+}
```

Pylint (4.0.2)

[warning] Statement seems to have no effect

(W0104)